src/EnergyPlus/DaylightingManager.cc

```diff
@@ -1846,20 +1744,23 @@ void FigureDayltgCoeffsAtPointsSetupForWindow(
 
             //            ! Error message if ref pt is too close to window.
             if (D1a > 0.0 && D1b > 0.0 && D1b <= HW && D1a <= WW) {
-                ShowSevereError(state,
-                                format("CalcDaylightCoeffRefPoints: Daylighting calculation cannot be done for zone {} because reference point "
-                                       "#{} is less than 0.15m (6\") from window plane {}",
-                                       Zone(ZoneNum).Name,
-                                       iRefPoint,
-                                       state.dataSurface->Surface(IWin).Name));
+                ShowSevereError(
+                    state,
+                    format("CalcDaylightCoeffRefPoints: Daylighting calculation cannot be done for Daylighting:Controls={} because reference point "
+                           "#{} is less than 0.15m (6\") from window plane {}",
+                           state.dataDaylightingData->daylightControl(daylightCtrlNum).Name,
+                           iRefPoint,
+                           state.dataSurface->Surface(IWin).Name));
```

Comment:
You changed from Zone(ZoneNum).Name to state.dataDaylightingData->daylightControl(daylightCtrlNum).Name ??